connectors/src/connectors/zendesk/lib/sync_ticket.ts

```diff
@@ -175,7 +175,16 @@ ${metadata}
 Conversation:
 ${comments
   .map((comment) => {
-    const author = users.find((user) => user.id === comment.author_id);
+    let author;
+    try {
+      author = users.find((user) => user.id === comment.author_id);
+    } catch (e) {
+      logger.warn(
+        { connectorId, e, ...loggerArgs },
+        "[Zendesk] Error finding the author of a comment."
+      );
+      author = null;
+    }
```

Comment:
```suggestion
   const author = users?.find((user) => user?.id === comment?.author_id) || null;
```

Reply:
that's not really what we're trying to fix with our try catch, will safely ignore for now

Reply:
Yeah actually not sure what this is supposed to do ?

Reply:
if there's an error here then it's a typescript error no ? there's no async at all